google/api_core/grpc_helpers_async.py

```diff
@@ -94,7 +100,28 @@ def __init__(self):
 
     async def read(self) -> P:
         try:
-            return await self._call.read()
+            result = await self._call.read()
```

Comment:
IIUC, this gRPC doing an actual read from the network, so logging it as such below makes sense.


(just pointing this out to contrast with my other comments about logging returning previously streamed messages to the user)